Tests/KituraTests/TestContentType.swift

```diff
@@ -71,4 +69,22 @@ class TestContentType: XCTestCase {
         XCTAssertEqual(result, "text/html")
     }
 
+    func testIsContentType() {
+        var result = contentType.isContentType("application/json", ofType: "json")
```

Comment:
You didn't add this test to the allTests variable up on top.

When unit testing tests please double check that the number of tests run on Linux equals the number of tests run on macOS.